File: problems/algorithms/string-rna-transcription/ruby/response/print/rna_transcription.print.rb
require_relative "../src/rna_transcription"

# RNA

# transcribing cytosine to guanine
p DNA.to_rna('C');
p 'G';

# transcribing guanine to cytosine
p DNA.to_rna('G');
p 'C';

# transcribing adenine to uracil
p DNA.to_rna('A');
p 'U';

# transcribing thymine to adenine
p DNA.to_rna('T');
p 'A';

# transcribing all dna nucleotides to their rna complements
p DNA.to_rna('ACGTGGTCTTAA');
p 'UGCACCAGAAUU';
 

File: src/pages/problems/_string-rna-transcription/_codes/js/response/rna-transcription.print.js
import { toRna } from "./rna-transcription.js";

// RNA

// transcribing cytosine to guanine
console.log(toRna("C"));
console.log("G");

// transcribing guanine to cytosine
console.log(toRna("G"));
console.log("C");

// transcribing adenine to uracil
console.log(toRna("A"));
console.log("U");

// transcribing thymine to adenine
console.log(toRna("T"));
console.log("A");

// transcribing all dna nucleotides to their rna complements
console.log(toRna("ACGTGGTCTTAA"));
console.log("UGCACCAGAAUU");
